sonar-plugin-api/src/main/java/org/sonar/api/batch/debt/DebtCharacteristic.java

```diff
@@ -23,7 +23,9 @@
 
 /**
  * @since 4.3
+ * @deprecated since 5.1 debt model will soon be unavailable on batch side
  */
+@Deprecated
 public interface DebtCharacteristic {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)